external_script_samples.py

```diff
@@ -0,0 +1,15 @@
+from depthai import CameraControl
+
+def stillCapture():
+    ctrl = CameraControl()
+    ctrl.setCaptureStill(True)
+    while True:
+        node.io['out'].send(ctrl)
+
+def stillCaptureEverySecond():
```

Comment:
We can add node as the first input to the function (and that'll make the linter happy as well)